tests/foreman/cli/test_setting.py

```diff
@@ -146,7 +151,7 @@ def test_positive_update_login_page_footer_text_without_value(self):
         login_text = Settings.list({'search': 'name=login_text'})[0]
         self.assertEqual('', login_text['value'])
 
-    @tier1
+    @tier2
```

Comment:
If test has importance Low, it has to be on Tier3 or Tier4.

Reply:
There is an update in CaseLevel, this might leads to tier2.

Reply:
Right, functional test on Component level belongs to Tier2, no matter the Importance.
Sometimes I get confused by these tier rules myself ;) .